src/utils.js

```diff
@@ -148,3 +148,49 @@ export function sanityCheck(
 
   return null;
 }
+
+export function getCryptoPassword(password: string) {
+  return `{SHA}'${crypto
+    .createHash('sha1')
+    .update(password, 'binary')
+    .digest('base64')}`;
+}
+
+/**
+ * changePasswordToHTPasswd - change password for existing user
+ * @param {string} body
+ * @param {string} user
+ * @param {string} passwd
+ * @param {string} newPasswd
+ * @returns {string}
+ */
+export function changePasswordToHTPasswd(
+  body: string,
+  user: string,
+  passwd: string,
+  newPasswd: string
+): string {
```

Comment:
Shouldn't it return an object?

Reply:
It returns a string, the whole HTPasswd body content as string.

Reply:
ok.. 👍